pacts/cli-lobby.json

```diff
@@ -0,0 +1,48 @@
+{
+  "consumer": {
+    "name": "CLI"
+  },
+  "provider": {
+    "name": "Lobby"
+  },
+  "interactions": [
+    {
+      "description": "A request to create a lobby",
+      "providerState": "The server is online",
+      "request": {
+        "method": "POST",
+        "path": "/lobby",
+        "headers": {
+          "Content-Type": "application/json"
```

Comment:
What's this?

Reply:
The request for which the server has to give the response mentioned below.
The content-type is json because that's generally the standard for POST requests, but considering the request body is empty, it doesn't really matter much.